src/extension.ts

```diff
@@ -13,13 +13,15 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
         vscode.window.showErrorMessage("Neovim: configure the path to neovim and restart the editor");
         return;
     }
+    const isWindows = process.platform == "win32";
+
     const highlightConfIgnore = settings.get("highlightGroups.ignoreHighlights");
     const highlightConfHighlights = settings.get("highlightGroups.highlights");
     const highlightConfUnknown = settings.get("highlightGroups.unknownHighlight");
     const mouseVisualSelection = settings.get("mouseSelectionStartVisualMode", false);
     const useCtrlKeysNormalMode = settings.get("useCtrlKeysForNormalMode", true);
     const useCtrlKeysInsertMode = settings.get("useCtrlKeysForInsertMode", true);
-    const useWsl = settings.get("useWSL", false);
+    const useWsl = settings.get("useWSL", false) && isWindows;
```

Comment:
I'm not very good at JavaScript, but shouldn't we put in a cheaper check first?

Reply:
Good point. I've swapped these now.